Sources/KSCrashRecording/include/KSCrashReportWriterCallbacks.h

```diff
@@ -0,0 +1,61 @@
+//
+//  KSCrashReportWriterCallbacks.h
+//
+//  Created by Gleb Linnik on 2025-08-17.
+//
+//  Copyright (c) 2012 Karl Stenerud. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall remain in place
+// in this source code.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#ifndef HDR_KSCrashReportWriterCallbacks_h
+#define HDR_KSCrashReportWriterCallbacks_h
+
+#include "KSCrashNamespace.h"
+
+#ifndef NS_SWIFT_UNAVAILABLE
+#define NS_SWIFT_UNAVAILABLE(_msg)
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Callback type for when a crash report is being written.
+ *
+ * @param policy The policy under which the report was written.
+ * @param writer The report writer.
+ */
+typedef void (*KSReportWriteCallbackWithPolicy)(struct KSCrash_ExceptionHandlingPolicy policy,
+                                                const KSCrashReportWriter *_Nonnull writer)
```

Comment:
Had to declare it in a separate file to suppress the compiler warning about missing nullability annotations in ReportWriter.